vectorstores/cloudsql/loader/documentloader.go

```diff
@@ -0,0 +1,441 @@
+package loader
+
+import (
+	"context"
+	"database/sql"
+	"encoding/csv"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"golang.org/x/exp/slices"
+	"regexp"
+	"strings"
+
+	"github.com/averikitsch/langchaingo/schema"
+	"github.com/averikitsch/langchaingo/textsplitter"
+	"github.com/averikitsch/langchaingo/util/cloudsqlutil"
+	"github.com/jackc/pgtype"
+)
+
+const (
+	defaultMetadataJSONColumn = "langchain_metadata"
+	defaultSchemaName         = "public"
+)
+
+// Document represents a loaded document with content and metadata.
+type Document struct {
```

Comment:
Do not redefine Documents - use https://github.com/averikitsch/langchaingo/blob/main/schema/documents.go#L4